### PR TITLE
mds: do not dispatch aborted internal requests

### PR DESCRIPTION
**Pull request by batrick**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [x] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/mds/MDSDaemon.cc [hunk 1]
```diff
@@ -149,6 +149,13 @@ void MDSDaemon::asok_command(
   if (command == "status") {
     dump_status(f);
     r = 0;
+  } else if (command == "sleep") {
```
Comment (leonid-s-usov) [inline src/mds/MDSDaemon.cc:152]: This is a great debugging tool. However, `sleep` might be a little too soft/generic for this command. How about `lock` or `stall`?

Comment (batrick) [reply src/mds/MDSDaemon.cc:152]: how about lockup?

**Review by leonid-s-usov — APPROVED** [review]

**Comment by batrick** [comment]
```
diff --git a/qa/tasks/cephfs/test_quiesce.py b/qa/tasks/cephfs/test_quiesce.py
index dadb11176a6..bdb803e8c47 100644
--- a/qa/tasks/cephfs/test_quiesce.py
+++ b/qa/tasks/cephfs/test_quiesce.py
@@ -666,8 +666,7 @@ class TestQuiesceMultiRank(QuiesceTestCase):
 
         def find_quiesce(blocked_on_remote_auth_pin):
             # verify no quiesce ops
-            ops = self.fs.get_ops(locks=False, rank=0, path=f"/tmp/mds.0-ops", status=status)['ops']
-            found = False
+            ops = self.fs.get_ops(locks=False, rank=0, path="/tmp/mds.0-ops", status=status)['ops']
             for op in ops:
                 type_data = op['type_data']
                 flag_point = type_data['flag_point']
```

tox fixes

**Comment by batrick** [comment]
jenkins test make check arm64

**Pull request merged**